### PR TITLE
Get article

### PR DESCRIPTION
**Pull request by GiovanniGobbs**
Get article + tests

### CODE REVIEW

File: utb/models.py [hunk 1]
```diff
@@ -8,18 +8,36 @@ class User(models.Model):
     email = models.CharField(max_length=30, unique=True)
     n_reports = models.IntegerField(default=0)
 
+    def as_dict(self):
```
Comment (matteofiordarancio) [inline utb/models.py:11]: Have you tried `self.__dict__.keys()`?

Comment (matteofiordarancio) [reply utb/models.py:11]: https://stackoverflow.com/questions/109087/how-to-get-instance-variables-in-python

File: utb/models.py [hunk 2]
```diff
@@ -8,18 +8,36 @@ class User(models.Model):
...
+        return {"name": self.name, "surname": self.surname, "email": self.email, "n_reports": self.n_reports}
+
 
 class Website(models.Model):
-    id = models.CharField(max_length=30, primary_key=True)
+    id = models.CharField(max_length=64, primary_key=True)  # hash of the name
+    name = models.CharField(max_length=100, unique=True)
+
+    def as_dict(self):
```
Comment (matteofiordarancio) [inline utb/models.py:19]: see above

File: utb/models.py [hunk 3]
```diff
@@ -8,18 +8,36 @@ class User(models.Model):
...
+    )
+    id = models.CharField(max_length=64, primary_key=True)  # hash of the url
+    url = models.CharField(max_length=300, unique=True)
+    name = models.CharField(max_length=300)
+    website = models.ForeignKey(Website, on_delete=models.CASCADE)
+    positive_reports = models.IntegerField(default=0)
+    negative_reports = models.IntegerField(default=0)
+    report_value = models.CharField(max_length=1, choices=REPORT_VALUES, default="L")
+
+    def as_dict(self):
```
Comment (matteofiordarancio) [inline utb/models.py:36]: see above

File: utb/views.py [hunk 4]
```diff
@@ -17,10 +21,95 @@ def add_user(request):
...
+    positive_reports = 0
+    negative_reports = 0
+
+    website_id = utils.hash_digest(website_name)
+    qs = Website.objects.filter(id=website_id)
+    if len(qs) == 0:
+        website = add_website(website_id, website_name)
+    else:
+        website = qs[0]
+
```
Comment (matteofiordarancio) [inline utb/views.py:91]: `website =  add_website(website_id, wesite_name) if len(qs) == 0 else qs[0]`

**Review by matteofiordarancio — CHANGES_REQUESTED** [review]

File: utb/models.py (hunk 1 above)
Comment (GiovanniGobbs) [reply utb/models.py:11]: The problem with that is that I don't want every field of the class in the dict

**Review by matteofiordarancio — APPROVED** [review]

**Pull request merged**